report.js

```diff
@@ -12,7 +12,8 @@ const scarfLibName = '@scarf/scarf'
 const privatePackageRewrite = '@private/private'
 const privateVersionRewrite = '0'
 
-const rootPath = path.resolve(__dirname).split('node_modules')[0]
+const rootPath = process.env.INIT_CWD
```

Comment:
let's keep in mind we changed this here

Reply:
Yeah, I actually found the cases where the old way breaks - not the cleanest to lump it in with this change, admittedly.